videos/WoT-Explainer-Video/WOTMotion.de.vtt

```diff
@@ -0,0 +1,251 @@
+WEBVTT
+
+1
+00:00:00.500 --> 00:00:03.000
+Ein Obstbauer sucht nach einer Möglichkeit 
+die Produktivität zu steigern
+
+2
+00:00:03.500 --> 00:00:06.000
+und gleichzeitig eine hervorragende 
+Produktqualität seiner Ernte zu garantieren.
+
+3
+00:00:06.500 --> 00:00:08.000
+Zur Überwachung und Kontrolle 
+der Wachstumsbedingungen,
+
+4
+00:00:08.500 --> 00:00:13.000
+plant der Landwirt Feuchtigkeits- und Temperatursensoren sowie 
+ein Sprinklersystem zu installieren.
+
+5
+00:00:13.500 --> 00:00:16.000
+Wettervorhersagedaten 
+von einem Online-Webdienst
+
+6
+00:00:16.500 --> 00:00:20.000
+wären auch nützlich, um den 
+Bewässerungsbedarf vorherzusagen.
+
+7
+00:00:20.500 --> 00:00:25.000
+Der Landwirt beschließt, einen 
+IT-Berater zu engagieren.
+
+8
+00:00:25.500 --> 00:00:29.000
+Er möchte die Entwickler seines bisherigen Internetauftritts nutzen 
+um eine IoT-Lösung zu realisieren.
+
+9
+00:00:29.500 --> 00:00:33.000
+Die Entwickler der Webagentur 
+freuen sich auf die neue Herausforderung
+
+10
+00:00:29.500 --> 00:00:36.000
+und sind bereit, ihr IoT-Wissen 
+ihr IoT-Wissen zu erweitern.
+
+11
+00:00:36.500 --> 00:00:41.000
+Sie sind Experten sowohl für Web-Frontend- als auch 
+Backend-Technologien unter Verwendung von Node JS.
+
+12
+00:00:41.500 --> 00:00:45.000
+Allerdings sind sie besorgt über 
+die Interoperabilitätsprobleme,
+
+13
+00:00:45.500 --> 00:00:49.000
+die bei einem früheren IoT-Projekt 
+aufgetreten sind. 
+
+14
+00:00:49.500 --> 00:00:53.000
+Nach einer Online-Recherche 
+für eine gute Lösung,
+
+15
+00:00:53.500 --> 00:00:58.000
+finden die Entwickler Informationen 
+über das Web of Things, WoT.
```

Comment:
Shall we mention once the German term? a la

über das Web of Things (dt. Web der Dinge), WoT.